### ISSUE TITLE
`precompile` files are not included in npm package

### ISSUE THREAD

**Issue by lukewestby** [wayfair/tungstenjs#250]
## What I'm trying to do

Install the `tungstenjs` and use the `tungsten_template` webpack loader from within the `tungstenjs` installation, as outlined in the project's documentation.
## Expected result

Webpack is able to look up `tungsten_template` from the `node_modules/tungstenjs/precompile` directory
## Actual result

The `precompile` directory is not present in the installation's directory. Only `dist` is available below the top level.
## Steps to reproduce

```
$ mkdir tungsten-install-test && cd tungsten-install-test
$ npm init --yes
$ npm install --save tungstenjs
$ ls node_modules/tungstenjs
```
---

Something that stands out in the installed `package.json` within tungsten's install directory is that the `files` property looks like

```
{
  // ...
  "files": [
    "dist"
  ],
  // ...
}
```

but I can't seem to figure out what configuration is causing this. Please let me know if more information is needed!

**Comment by ndreckshage** [comment]
This fixes https://github.com/wayfair/tungstenjs/pull/251 - tested on scoped npm module (@ndreckshage/tungstenjs)

**Comment by andrewrota** [comment]
Would the template compiler APIs exposed [here](https://github.com/wayfair/tungstenjs/blob/master/adaptors/shared/common_exports.js#L21-L23)  work?  I use `precompiler` as a [webpack template loader](https://github.com/andrewrota/boilerplate-tungstenjs-app/blob/master/loaders/template_loader/index.js) in my boilerplate tungsten apps.

The published npm package is intentionally only the dist files to prevent deep-linking into the repo when consuming the library (also since including a non-compiled build would require consuming libraries to use the same versions of loaders like babel).  Custom builds can be done by cloning the repo and installing its dev dependencies.

If there's something we're not exposing that is necessary to consume tungsten then the best way to handle that would be to expose it on the tungsten API itself.

And yes, I know all of this should be updated in our docs.  I'm hoping to do a significant update of the readme soon.

**Comment by lukewestby** [comment]
Perfect! I can submit a PR to update the docs if you'd like.

**Comment by andrewrota** [comment]
That'd be awesome. I'm unable to do much until next week but I can spend some time on them then.

**Comment by artemruts** [comment]
Closing, looks like primary problem was addressed with #255.
